File: pdfsecondmonitor/FileViewParam.vb


Public Class FileViewParam
    Public Property FileName As String
    Public Property zoom As Decimal
    Public Overrides Function ToString() As String
        Return $"{System.IO.Path.GetFileName(FileName)}"

    End Function

    Public Sub New(ByVal filename As String)
        Me.FileName = filename
        Me.zoom = 100
    End Sub

    Public Sub New()
        'シリアル化用
    End Sub
End Class

File: MediaToSecondMonitor/FileViewParam.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MediaToSecondMonitor
{
    public class FileViewParam
    {
        public string FileName { get; set; }
        public decimal zoom { get; set; } = 100;
        public override string ToString()
        {
            return $"{System.IO.Path.GetFileName(FileName)}";
        }

    }
}
